platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -150,23 +149,38 @@ open class FeatureFragment : Fragment() {
                 foregroundResources.icons.getValue(actionButton.action)
             )
             buttonView.setOnClickListener {
-                if (actionButton == ActionButton.CLOSE)
+                if (actionButton == ActionButton.CLOSE) {
                     navigateBack()
-                else
+                } else {
```

Comment:
Are these linting errors that have been fixed?

Reply:
automatic linter fix yep